_rules/video-only-description-track-ac7dc6.md

```diff
@@ -99,8 +99,8 @@ A video element with audio.
 
 ```html
 <video controls>
-	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/video.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/video.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/video.webm" type="video/webm">
 	<track kind="descriptions" src="rabbit-video-descriptions.vtt" />
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`